src/api/controllers/db-controller.ts

```diff
@@ -170,9 +171,7 @@ export function getTxStatus(txStatus: DbTxStatus | string): string {
 
 type HasEventTransaction = SmartContractTransaction | ContractCallTransaction;
 
-export function getEventTypeString(
-  eventTypeId: DbEventTypeId
-): ElementType<Exclude<HasEventTransaction['events'], undefined>>['event_type'] {
+export function getEventTypeString(eventTypeId: DbEventTypeId) {
```

Comment:
Can you add the new return type?

Reply:
There isn't any new return type. However, there are no `events` in the `BaseTransaction`. So, I had to remove the return type from here. 
I have updated the return type, let me know if that looks fine.